db/queries/base.py

```diff
@@ -176,4 +190,7 @@ def __init__(
 
     @property
     def is_base_column(self):
+        """
+        A base column is a column on a query's base table.
```

Comment:
Using our terminology so far, it's more properly an _initial_ column (of the query) which is also in the base table. A non-query base table column shouldn't be included here.

Reply:
Good catch.